scipy/interpolate/_ppoly.pyx

```diff
@@ -94,7 +92,7 @@ def evaluate(double_or_complex[:,:,::1] c,
         if i < 0:
             # xval was nan etc
             for jp in range(c.shape[2]):
-                out[ip, jp] = nan
+                out[ip, jp] = libc.math.NAN
```

Comment:
This seems to be the only file in the diff where this isn't imported à la `from libc.math cimport NAN`

Reply:
The code in this file was deliberately written like that by @pv (I'm guessing because of the `malloc`/`free` usage), so I'd much prefer not to change the style decision he made here. I chose to be consistent with the style in this file.